src/tests/Interop/IJW/CopyConstructorMarshaler/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ set(SOURCES IjwCopyConstructorMarshaler.cpp)
 # add the shared library
 add_library (IjwCopyConstructorMarshaler SHARED ${SOURCES})
 target_link_libraries(IjwCopyConstructorMarshaler ${LINK_LIBRARIES_ADDITIONAL})
```

Comment:
@jkoritzinsky @hoyosjs I hit some merge conflicts in this line in the 3 CMakeLists.txt files. I was careful not to alter this line. In `main`, it has a `PRIVATE` right after the `$`.